move-to-sdk/app/io/sphere/sdk/facets/FacetOptionMapper.java

```diff
@@ -0,0 +1,17 @@
+package io.sphere.sdk.facets;
+
+import java.util.List;
+
+/**
+ * Mapper class that allows to transform the facet options, as they are extracted from the search result, into another list.
+ * For example, to transform a list of category IDs into a hierarchical structure of categories with localized names.
+ */
+public interface FacetOptionMapper {
```

Comment:
Does it make sense to 

`@FunctionalInterface public interface FacetOptionMapper extends UnaryOperator<List<FacetOption>> {`?